message.go

```diff
@@ -264,6 +264,14 @@ func SetHeader(h map[string][]string) FileSetting {
 	}
 }
 
+// SetFilename is a file setting to set the name of the attachment if the name is different
```

Comment:
Please limit the line length to 80 characters and add a dot at the end of the sentence.

I think `Rename()` might be more clear. `SetFilename` can be confusing, some people could think you have to put the filename on disk here.